telemeta/views/enum.py

```diff
@@ -0,0 +1,124 @@
+from telemeta.views.core import *
+import telemeta.models
+
+class EnumView(object):
+    enu = []
+
+    def __init__(self):
+        self.enu = self.__get_enumerations_list()
+
+    def __get_admin_context_vars(self):
+        return {"enumerations": self.__get_enumerations_list()}
+
+    def enumerations(self, request):
+        return render(request, 'telemeta/enumerations.html', self.__get_admin_context_vars())
+
+    def enumeration (self, request, enumeration_id):
+        atr = "";
+
+        enumeration = self.__get_enumeration(enumeration_id)
+        if enumeration == None or enumeration.admin == "True":
+            raise Http404
+        vars = self.__get_admin_context_vars()
+        vars["enumeration_id"] = enumeration._meta.module_name
+        vars["enumeration_name"] = enumeration._meta.verbose_name
+        vars["enumeration_values"] = enumeration.objects.all()
+        vars["enumeration_support"] = ""
+        vars["enumeration_count"] = []
+        f = MediaCollection._meta.get_all_field_names()
+        for field in f:
+            if field in enumeration._meta.db_table.replace(" ", "_"):
+                atr = field;
+        if enumeration._meta.db_table.replace(" ", "_") == "context_keywords":
+            vars["enumeration_support"] = "Item"
+            vars["enumeration_count"] = self.__getCountKeyWord(vars["enumeration_values"])
+        else:
+            if atr == "":
+                vars["enumeration_support"] = "Item"
+                vars["enumeration_count"] = self.__getCountItem(enumeration, vars["enumeration_values"])
+            else:
+                vars["enumeration_support"] = "Collection"
+                vars["enumeration_count"] = self.__getCountColl(vars["enumeration_values"], atr)
+
+        return render(request, 'telemeta/enumeration.html', vars)
+
+
+    def set_enum_file(self,request):
```

Comment:
Good point to have the enumerations in the public home, but this design is really bad for several reasons:

1. we usually avoid the app to write any dynamic data in its own code directory (so that it is **never** commited with the code). It can be fixtures, but with static data..
2. it doesn't ensure concurrency write access during editing
3. the DB is made for that

So, we could really prefer to have this enum options as an Enumeration class property.